test/runtests.jl

```diff
@@ -348,7 +348,7 @@ end
 @testset "Linear model with QR method and NASTY data" begin
     x = [1, 2, 3, 4, 5, 6, 7, 8, 9]
     nasty = DataFrame(X = x, TINY = 1.0E-12*x)
-    mdl = lm(@formula(X ~ TINY), nasty; method=:qr)
+    mdl = lm(@formula(X ~ TINY), nasty)
```

Comment:
Have you checked that Cholesky is covered well enough after this PR? Haven't some tests for QR become redundant now that it's the default?

Reply:
Yes. Most of the testsets loop through both methods.